dependencies_ros1.rosinstall

```diff
@@ -1,3 +1,6 @@
 # PCL ros tools, this is required because using custom version of PCL and VTK
+# As of ROS Noetic, the default PCL and VTK versions are sufficient to no longer require custom installs
 - git: {local-name: perception_pcl, uri: 'https://github.com/ros-perception/perception_pcl.git', version: 1.7.1}
+
+# As of ROS Noetic, the following command may be used instead: `sudo apt install ros-noetic-ros-industrial-cmake-boilerplate`
```

Comment:
Maybe for clarity it's worth creating separate kinetic/melodic/noetic files

Reply:
I'm not sure I agree.  The requirements are the same for Kinetic and Melodic, and there are no requirements for Noetic.  In general, I feel it would be better to drop a kinetic-devel or melodic-devel branch and then move along, rather than trying to maintain compatibility for three different environments by duplicating configuration files.  That's what most repositories do.

Additionally, according to Ubuntu's [website](https://ubuntu.com/about/release-cycle), 16.04 (and therefore Kinetic) are only supported for security functions as of April 30, 2021.  How long are we anticipating continued Kinetic support?